dev-php/fpdf/fpdf-1.86.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# shellcheck disable=SC2034
+EAPI=8
+
+MY_PV="$(ver_rs 1 '')"
+MY_P="${PN}${MY_PV}"
+
+DESCRIPTION="FPDF is a PHP class which allows to generate PDF files with pure PHP"
+HOMEPAGE="http://www.fpdf.org/"
+SRC_URI="http://www.fpdf.org/en/dl.php?v=${MY_PV}&f=tgz -> ${MY_P}.tgz"
+
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="freedist"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```

Comment:
Has this been tested on x86? I believe keywords should only be added for arches that the package has been tested on, even for unstable keyword.

Reply:
Deleted `~x86`